plasma_framework/test/src/framework/utils/ExitPriority.test.js

```diff
@@ -5,6 +5,8 @@ const { expect } = require('chai');
 
```

Comment:
There is typo here https://github.com/omisego/plasma-contracts/blob/8e21daf3752e47a537d745fc3d51df5057f96e86/plasma_framework/test/src/framework/utils/ExitPriority.test.js#L15